learning_journal/templates/posts.jinja2

```diff
@@ -188,10 +188,10 @@
     <!-- /.container -->
 
     <!-- jQuery -->
-    <script src="js/jquery.js"></script>
+    <script src="../static/js/jquery.js"></script>
```

Comment:
Consider adding this into your base template instead of importing it into every template.